recipe/meta.yaml

```diff
@@ -119,8 +121,10 @@ requirements:
     - mkl {{ mkl }}.*                 # [blas_impl == "mkl"]
```

Comment:
please replace {{ mkl }}, {{ cudnn }}. {{ cudatoolkit }} by real version.  Don't use jinja2 stuff here from cbc.
mkl has version pinning 2021.*
cudnn and cudatoolkit need to be mentioned explicit.  I see that its definition was comment in the local conda_build_config.yaml, which is not correct

Reply:
The commented out section is for `pytorch_variant == "gpu"`. These definitions aren't needed unless we select the gpu variant.